server/backend/cron/fileDownlad.go

```diff
@@ -0,0 +1,88 @@
+package cron
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/TUM-Dev/Campus-Backend/model"
+	"github.com/disintegration/imaging"
+	"github.com/gabriel-vasile/mimetype"
+	"github.com/getsentry/sentry-go"
+	log "github.com/sirupsen/logrus"
+	"gorm.io/gorm"
+	"image"
+	"io/ioutil"
+	"net/http"
+	"strings"
+)
+
+//fileDownloadCron Downloads all files that are not marked as finished in the database.
+func (c *CronService) fileDownloadCron() error {
+	var files []model.Files
+	err := c.db.Find(&files, "downloaded = 0").Scan(&files).Error
```

Comment:
Oh my, funny that mysql still doesn't support boolean types properly. This is already stupid, that you have to use a `= 0` up here but then when updating the model below you can use a proper `true` - oh well, guess we have to live with it.